database/fingertips-db/PostDeployment/PopulateFingertips.sql

```diff
@@ -346,20 +347,33 @@ INSERT INTO [dbo].[HealthMeasure]
     Year
 )
 SELECT
-    (SELECT TOP 1 [AreaKey] FROM [dbo].[AreaDimension] WHERE [Code] = LTRIM(RTRIM(temp.AreaCode))),
-    (SELECT TOP 1 [IndicatorKey] FROM [dbo].[IndicatorDimension] WHERE IndicatorId = temp.IndicatorId),
-    (SELECT TOP 1 [SexKey] FROM [dbo].[SexDimension] WHERE [Name] = LTRIM(RTRIM(temp.Sex))),
-    (SELECT TOP 1 [AgeKey] FROM [dbo].[AgeDimension] WHERE [AgeID] = temp.AgeID),
-    (SELECT TOP 1 [DeprivationKey] FROM [dbo].[DeprivationDimension] WHERE [Name] = LTRIM(RTRIM(temp.Category)) AND [Type]=LTRIM(RTRIM(temp.CategoryType))),
+    areadim.AreaKey,
+    inddim.[IndicatorKey],
+    sexdim.[SexKey],
+    agedim.[AgeKey],
+    depdim.[DeprivationKey],
     Count,
     Denominator,
     Value,
     Lower95CI,
     Upper95CI,
     Year
-FROM #TempHealthData temp
+FROM 
+	#TempHealthData temp
+JOIN
+	[dbo].[AreaDimension] areadim ON LTRIM(RTRIM(temp.AreaCode))=areadim.[Code]
```

Comment:
```suggestion
	[dbo].[AreaDimension] areadim ON LTRIM(RTRIM(temp.AreaCode)) = areadim.[Code]
```

Minor but could add the space. Same for end of  line 372.